src/api/java/baritone/api/utils/ExampleBaritoneControl.java

```diff
@@ -711,4 +717,11 @@ private Goal parseGoal(String[] params) {
         }
         return goal;
     }
+
+    public DimensionType getDimensionByName(String nameIn){ String name = nameIn.toLowerCase();
+        if("end".contains(name)) return DimensionType.THE_END;
```

Comment:
does not follow baritone style -- we do not allow `if`s without `{`

Reply:
oh alright